docs/platform-setup.md

```diff
@@ -18,3 +18,126 @@ For each job, disable the GitHub multibranch status:
 - Select "Disable GitHub Notifications".
 
 - Click on "Save".
+
+## Configure Nexus Docker Registry
+
+### Prerequisite
+
+- Nexus is installed.
+- NGINX Ingress Controller is installed.
+
+Configuring docker registry with nexus should follow this steps:
+
+### Create New Docker Repository
+
+Open the platform team's [Nexus](https://nexus.platform.dev.nuxeo.com/). Start new session using admin user
+
+#### Blob Storage And Repository Configuration
+
+- Create a blob storage : `docker-registry`
+
+  - In Server Administration and Configuration > Blob Stores,  click on the "Create blob store" button
+
+  - Complete all required fields and click on "Save"
+
+- Create a docker (hosted) nexus repository: `docker-registry`
+
+  - In Server Administration and Configuration > Repositories,  click on the "Create repository" button
+
+  - On storage section, use the previously created blob store
+
+  - Complete all required fields and click on "Save"
+
+### Cleanup Policy Configuration
+
+For each nexus docker registry, create a new policy cleanup and apply to it.
+
+#### Create New Cleanup Policy
+
+- In Server Administration and Configuration > Cleanup Policies,  click on the "Create Cleanup Policy" button
+
+- To keep all nuxeo builder and jenkins master images ( latest tag for example ) add this asset name matcher `~(.*nuxeo/builder-.*/manifests/latest|.*nuxeo/platform.*-jenkinsx/manifests/[0-9]{1,}.[0-9]{1,}.[0-9]{1,})`
+
+- Complete all required fields and click on "Save"
+
+#### Apply Cleanup Policy
+
+- Open an existing nexus repository configuration for instance [docker-registry](https://nexus.platform.dev.nuxeo.com/#admin/repository/repositories:docker-registry)
+
+- Go done to cleanup section and select the corresponding item
+
+- Click on "Save".
+
+### Create New Docker Ingress Service
+
+This ingress service will act as Reverse Proxy for nexus docker repository.
+The reverse proxy would direct docker commands ( pull, push or login ) received at
+
+`https://docker.platform.dev.nuxeo.com`
+
+to
+
+`https://nexus.platform.dev.nuxeo.com/repository/docker-registry/`
+
+Create an ingress configuration file following this [example](../templates/docker-ingress.json)
+
+Create new ingress service from the configuration file
+
+```bash
+kubectl create -f templates/docker-ingress.json
+```
+
+Check if the ingress service is ready
+
+```bash
+kubectl get -f templates/docker-ingress.json
+```
+
+List the catalog from the new docker registry under nexus
+
+```bash
+DOCKER_REGISTRY=$(kubectl get -f templates/docker-ingress.json -o json --output=jsonpath={.spec.rules[].host} )
+curl https://${DOCKER_REGISTRY}/v2/_catalog
+```
+
+### Update The Config.json Secret
```

Comment:
Maybe we could use something more functional here, instead of "Update The Config.json Secret", "Update the Jenkins X Docker Configuration".

And, as an introduction in this section, we could add a small paragraph explaining that we need to add authentication for the newly created Docker registry to allow pushing images.

Reply:
changes applied